packages/client/src/lib.js

```diff
@@ -112,15 +113,26 @@ class NFTStorage {
      * @readonly
      */
     this.rateLimiter = rateLimiter || createRateLimiter()
+
+    /**
+     * @readonly
+     */
+    this.did = did
   }
 
   /**
    * @hidden
-   * @param {string} token
+   * @param {object} options
+   * @param {string} options.token
+   * @param {string} [options.did]
    */
-  static auth(token) {
+  static auth({ token, did }) {
```

Comment:
Signature changed but none of the callers

Reply:
That is because I use spread to capture both e.g see line 170

Reply:
Oh yes my apologies!